feat: blockchain agnostic inputs (#89)
cartesi/rollups-contracts: onchain/.changeset/cuddly-ears-obey.md

---
"@cartesi/rollups": minor
---

> Fields in the `InputAdded` event were renamed. `inputIndex` was renamed to `index`, and `input` was renamed to `payload`.

Reviewer: I know I called them fields, but after reading the Solidity documentation, it seems that they are normally called arguments/parameters. The distinction is the same with functions: arguments are values (like `123` in `f(123)`), while parameters are variables (like `x` in `function f(x) end`).

Also, it seems that the right way to say is "renamed as".

```suggestion
Two parameters of the `InputAdded` event were renamed: `inputIndex` was renamed as `index`, and `input` was renamed as `payload`.
```
